src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -33,28 +33,39 @@ public class MeterProviderSdk
         private readonly object collectLock = new object();
         private readonly MeterListener listener;
         private readonly List<MetricProcessor> metricProcessors = new List<MetricProcessor>();
+        private readonly List<MetricReader> metricReaders = new List<MetricReader>();
         private int metricIndex = -1;
 
         internal MeterProviderSdk(
             Resource resource,
             IEnumerable<string> meterSources,
             List<MeterProviderBuilderSdk.InstrumentationFactory> instrumentationFactories,
-            MetricProcessor[] metricProcessors)
+            MetricProcessor[] metricProcessors,
```

Comment:
Looks like MetricProcessor is not used anymore. Is the plan to remove in a follow up?

Reply:
yes a lot of cleanup coming..